### ISSUE TITLE
fr-data-safe-input input doesn't have a label

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#2225]
This is a problem per WCAG2 1.3.1 (A), reported as E885 by TotalValidator. Making the input a `type="hidden"` is enough to solve the problem.
